modules/CmpdReg/src/server/routes/CmpdRegRoutes.coffee

```diff
@@ -297,8 +370,32 @@ exports.fileSave = (req, resp) ->
 	req.pipe(request[req.method.toLowerCase()](cmpdRegCall)).pipe(resp)
 
 exports.metaLots = (req, resp) ->
```

Comment:
nit: This function name is terrible. Can we rename it to "createMetaLots" or "updateMetaLots" or "putMetaLots"? Anything besides just `metaLots`

Reply:
I changed it to exports.saveMetaLot because it is a singular meta lot that gets saved.